### PR TITLE
Udp adapter for logger

### PR DESCRIPTION
**Pull request by vitalypanait**
Use UDP protocol for send logs to external server

### CODE REVIEW

**Comment by sergeyklay** [comment]
We don't add changes directly to stable branches or `master`.

**Comment by sergeyklay** [comment]
Also could you please update `README.md` from root of project?

File: Library/Phalcon/Logger/Adapter/Udplogger.php [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+use Phalcon\Logger\Adapter as LoggerAdapter;
+use Phalcon\Logger\AdapterInterface;
+
+/**
+ * Phalcon\Logger\Adapter\Udplogger
+ * Sends messages using UDP protocol to external server
+ *
+ * @version 0.1
+ * @author  Vitaliy Panait <panait.vi@gmail.com>
+ * @license The BSD 3-Clause License {@link http://opensource.org/licenses/BSD-3-Clause}
```
Comment (sergeyklay) [inline Library/Phalcon/Logger/Adapter/Udplogger.php]:
```
<?php
/*
  +------------------------------------------------------------------------+
  | Phalcon Framework                                                      |
  +------------------------------------------------------------------------+
  | Copyright (c) 2011-2016 Phalcon Team (http://www.phalconphp.com)       |
  +------------------------------------------------------------------------+
  | This source file is subject to the New BSD License that is bundled     |
  | with this package in the file docs/LICENSE.txt.                        |
  |                                                                        |
  | If you did not receive a copy of the license and are unable to         |
  | obtain it through the world-wide-web, please send an email             |
  | to license@phalconphp.com so we can send you a copy immediately.       |
  +------------------------------------------------------------------------+
  | Authors: Vitaliy Panait <panait.vi@gmail.com>                          |
  +------------------------------------------------------------------------+
*/
```

File: Library/Phalcon/Logger/Adapter/Udplogger.php [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+     * @param array   $context
+     */
+    public function logInternal($message, $type, $time, $context = [])
+    {
+        $this->logs[] = [
+            'name'    => $this->name,
+            'message' => $message,
+            'type '   => $type,
+            'time'    => $time
+        ];
```
Comment (sergeyklay) [inline Library/Phalcon/Logger/Adapter/Udplogger.php]:
```php
$this->logs[] = compact('message', 'type', 'time', 'context');
```

**Comment by vitalypanait** [comment]
@sergeyklay Which branch should I use? Should I create new PR?

**Pull request closed without merging**